tests/unittests/TFLiteImporterTest.cpp

```diff
@@ -96,12 +97,27 @@ static void loadAndRunModel(std::string modelName, float maxError = 1e-6) {
   size_t inpIdx = 0;
   for (const auto &inpPH : inputPH) {
     std::string inpFilename = dataBasename + ".inp" + std::to_string(inpIdx++);
-    Tensor *inpT = bindings.get(inpPH);
-    loadTensor(inpT, inpFilename);
+
+    auto phTy = inpPH->getNthResult(0).getType();
+    // Load reference tensor.
+    Type newTy = *phTy;
```

Comment:
This code is hard to read. When the input is float we should load the data as before and when the data needs to be quantized we should read the data into another tensor (say inpFloatT) and quantize it into the final tensor inpT. Quantizing a tensor looks very strange. Same goes for the output path as well.

Reply:
There is no quantization in place. Output tensor is assigned, not pointed to. Please look at quantizeTensor function - temporary tensor is introduced, quantized, and then return.

Reply:
Added few more comments to explain the proces.